exec/java-exec/src/main/resources/rest/query/result.ftl

```diff
@@ -34,6 +34,9 @@
     <td align='left'>
       <button type="button"  title="Open in new window" onclick="popOutProfile('${model.getQueryId()}');" class="btn btn-default btn-sm">
       <b>Query Profile:</b> ${model.getQueryId()} <span class="glyphicon glyphicon-new-window"/></button>
+     </td>
+     <td>
+         <span class="input-group-addon" style="font-size:95%"><b>Query State:</b> ${model.getQueryState()}</span>
```

Comment:
It might be worth putting the Status in a Label with the colour indicative of the status.
Here is a good reference: https://www.w3schools.com/bootstrap/bootstrap_badges_labels.asp